source/fundamentals/connections/tls.txt

```diff
@@ -0,0 +1,212 @@
+.. _golang-tls:
+
+========================
+Enable and Configure TLS
+========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, security, connection options
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the TLS protocol to secure 
+your connection to a MongoDB deployment. To configure your connection to
+use TLS, enable the TLS option and provide your certificates for
+validation when creating a client.
+
+This guide includes the following sections:
+
+- :ref:`Enable TLS <golang-enable-tls>` describes ways to enable TLS on
+  your connection
+
+- :ref:`Configure Certificates <golang-configure-tls-certificates>`
+  describes the certificates required to configure TLS
+
+- :ref:`Reference Certificates in a Client <golang-client-tls-connect>`
+  provides an example of how to create a ``Config`` struct to configure your
+  TLS options
+
+- :ref:`Additional Information <golang-tls-addtl-info>`
+  provides links to resources and API documentation for types
+  and methods mentioned in this guide
+
+.. tip::
+   
+   To learn more about TLS, see the Wikipedia entry on
+   :wikipedia:`Transport Layer Security <w/index.php?title=Transport_Layer_Security&oldid=1184063676>`.
+
+.. _golang-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS on a connection to your MongoDB instance
+in one of the following ways:
+
+- Setting the ``tls`` option to ``true`` in your connection string
+- Passing an empty ``Config`` struct to the ``SetTLSConfig()``
+  method when creating a ``ClientOptions`` instance
+
+Select from the following :guilabel:`Connection String` and
+:guilabel:`ClientOptions` tabs to see a corresponding code sample:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection string tls true
+
+      .. code-block:: go
+         :emphasize-lines: 1
+
+         uri := "mongodb://<hostname>:<port>?tls=true"
+         opts := options.Client().ApplyURI(uri)
+         client, _ := mongo.Connect(context.TODO(), opts)
+
+   .. tab:: ClientOptions
+      :tabid: clientoptions tls true
+      
+      .. code-block:: go
+         :emphasize-lines: 2
+         
+         uri := "<connection string>"
+         opts := options.Client().ApplyURI(uri).SetTLSConfig(&tls.Config{})
+         client, _ := mongo.Connect(context.TODO(), opts)
+
+.. note::
+   
+   If your connection string uses a use a DNS SRV record by including
```

Comment:
I: typo here. do you think mention DNS SRV records here is useful? you could link out to the server connection docs instead.

```suggestion
   If your connection string uses a DNS SRV record by including
```

Reply:
We should mention this since it's based on [user feedback](https://jira.mongodb.org/browse/DOCSP-34060). The server docs mention it, but that information is not in a format that is quickly findable.